src/js/components/FormField/__tests__/FormField-test.tsx

```diff
@@ -158,6 +160,41 @@ describe('FormField', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test.only('does not apply focusIndicator when formField.focusIndicator is false', () => {
+    const { container } = render(
+      <Grommet
+        theme={{
+          formField: {
+            focus: {
+              nativeFocus: false,
```

Comment:
I know the name of this prop might change but just flagging so we don't forget to also change the name here.
```suggestion
              childFocus: false,
```